src/main/java/carbonbot/DukeException.java

```diff
@@ -0,0 +1,16 @@
+package carbonbot;
+
+/**
+ * Thrown to indicate that an exception occurred that is specific to Duke.
+ */
+public class DukeException extends Exception {
```

Comment:
If your chatbot is CarbonBot, shouldnt the exceptions be named something like CarbonBotException? (But if duke is fine then its alright too!)